ServiceStation/WebApp/Controllers/CarController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using BusinessLayer.Models.DTO;
+using BusinessLayer.Services.ServiceStationService;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace WebApp.Controllers
+{
+    public class CarController : Controller
+    {
+        private readonly CarService _carService;
+        private readonly ILogger<CarController> _logger;
+        
+        public CarController(CarService carService, ILogger<CarController> logger)
+        {
+            _carService = carService;
+            _logger = logger;
+        }
+        
+        public async Task<IActionResult> Index()
+        {
+            return View(await _carService.GetItems());
+        }
+        
+        public ActionResult Create()
+        {
+            return View();
+        }
+        
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Create(IFormCollection collection)
+        {
+            try
+            {
+                var car = new Car
+                {
+                    CarNumber = collection["CarNumber"],

```

Comment:
You can refactor this by creation a model that will be used in the form (or use ViewModel and other existing BLL model) and instead of `IFromCollection collection` use `[FromForm]CarViewModel car`.
But it's not necessary.